store/keystore.go

```diff
@@ -199,3 +208,41 @@ func saveToPem(out io.Writer, key openssl.PrivateKey) error {
 
 	return err
 }
+
+func loadPassphrase(passphrase_file string) (passphrase string, err error){
+	if passphrase_file != "" {
+		if passphrase_file == "-" {
+			log.Debugf("Read passphrase from pipe.")
+			fi, err := os.Stdin.Stat()
+			if err != nil {
+				log.Fatal(err)
+				return "", err
+			}
+			if fi.Mode() & os.ModeNamedPipe == 0 {
+			} else {
+				reader := bufio.NewReader(os.Stdin)
+				var output []rune
+				for {
+					input, _, err := reader.ReadRune()
+					if err != nil && err == io.EOF {
+						break
+					}
+					output = append(output, input)
+				}
+				passphrase = string(output[:len(output)-1])
+			}
```

Comment:
Why do we need to do it differently here? Can't we just open the right source (stdin or file) and then use `ioutil.ReadAll` for both?

Reply:
I'm not sure about that.
This is my first software in go and I only found this version.
Will try to unite this.